include/ibeo_core/ibeo_core.h

```diff
@@ -777,6 +822,28 @@ class ScanData2208 : public IbeoTxMessage
   std::vector<Point3DL> get_scan_points();
 };
 
+class ScanData2209 : public IbeoTxMessage
+{
+public:
+  static const int32_t DATA_TYPE;
+
+  NTPTime scan_start_time;
+  uint32_t scan_end_time_offset;
+  bool fused_scan;
+  MirrorSide mirror_side;
+  CoordinateSystem coordinate_system;
```

Comment:
There are a pretty good number of additional flags here too. Can you please add/parse them?

0x00000001: ground
0x00000002: dirt
0x00000004: rain
0x00000008: coverage
0x00000010: background
0x00000020: reflector
0x00000080: APD reduction active
0x00000100: upside down
0x00000200: scan fusion
0x00000400: mirror side, 0 = front, 1 = rear
0x00000800: scan point coordinate system: 0 = scanner coordinates, 1 = vehicle / reference coordinates
0x00001000: segment ID of scan points is used as object ID
0x00002000: ray angle corrected(window refraction, etc.)